pkg/api/routes.go

```diff
@@ -1873,22 +1873,22 @@ func (rh *RouteHandler) Logout(response http.ResponseWriter, request *http.Reque
 }
 
 // github Oauth2 CodeExchange callback.
-func (rh *RouteHandler) GithubCodeExchangeCallback() rp.CodeExchangeCallback {
+func GithubCodeExchangeCallback[C oidc.IDClaims](c *Controller) rp.CodeExchangeCallback[C] {
```

Comment:
Why was this change needed?

Reply:
zitadel v3 made tokens as a generic type, so this needs resolving.

Reply:
pkg/api/routes.go:1876:54: cannot use generic type rp.CodeExchangeCallback[C "github.com/zitadel/oidc/v3/pkg/oidc".IDClaims] without instantiation
pkg/api/routes.go:1878:11: cannot use generic type "github.com/zitadel/oidc/v3/pkg/oidc".Tokens[C "github.com/zitadel/oidc/v3/pkg/oidc".IDClaims] without instantiation
pkg/api/routes.go:1911:54: cannot use generic type rp.CodeExchangeUserinfoCallback[C "github.com/zitadel/oidc/v3/pkg/oidc".IDClaims, U rp.SubjectGetter] without instantiation
pkg/api/routes.go:1912:62: cannot use generic type "github.com/zitadel/oidc/v3/pkg/oidc".Tokens[C "github.com/zitadel/oidc/v3/pkg/oidc".IDClaims] without instantiation
pkg/api/routes.go:1915:17: info.GetEmail undefined (type "github.com/zitadel/oidc/v3/pkg/oidc".UserInfo has no field or method GetEmail)
pkg/api/routes.go:1925:19: info.GetClaim undefined (type "github.com/zitadel/oidc/v3/pkg/oidc".UserInfo has no field or method GetClaim)